Languages/es/03_Funcion_es/Function.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+contract FunctionTypes{
+    uint256 public number = 5;
+    
+    constructor() payable {}
+
+    // function type
+    // function (<parameter types>) {internal|external} [pure|view|payable] [returns (<return types>)]
+    // default function
+    function add() external{
+        number = number + 1;
+    }
+
+    // pure: not only does the function not save any data to the blockchain, but it also doesn't read any data from the blockchain.
```

Comment:
Should this be translated ? I did it in my chapter jajaja

Reply:
noted